adaptors/stripe.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Stripe Adaptor
+---
+
+## About Stripe
+
+[Stripe](https://stripe.com/) is a solution designed to automate payment
+processing and financial operations.
+
+## Integration Options
+
+Stripe has a REST API that enables external services like OpenFn to pull data
+from Stripe, or push data from external apps to Stripe. This option is suited
+for scheduled, bulk syncs or workflows that must update data in Stripe with
+external information. See [functions](/adaptors/packages/stripe-docs) for more
+on how to use this adaptor to work with the API.
+
+## Authentication
+
+See [Stripe docs](https://docs.stripe.com/api/authentication) for the latest on
+supported authentication methods. When integrating with Stripe via OpenFn, only
+one primary authentication method is supported.
+
+Api Key (requires api key created after authenticating in Stripe). See this
```

Comment:
I find this confusing `(requires api key created after authenticating in Stripe)`

Can we simplify the explanation to something like. 

>When integrating Stripe via OpenFn, Stripe API Key is required . [See our configuration docs](url) to learn more.